### DISCUSSION TITLE
terraform local-exce provisioner error

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1500] [Q&A] [answered]
hi,

when i am trying to give a provision to frontend, backend and mysql its throwing an error that failed to connect

![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/50f8b3b4-c40b-4f64-be38-5538d89f4354)

**Comment by sreedharm07** [comment]


**Reply by aambadkar** [reply]
but if i check my DNS id and server id are same. and before using terraform apply i destroy the before created servers

![tr1](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/c89d4952-4fa6-4559-ad07-b47bf0389345)
![tr2](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/61725f6a-eadd-48c9-9396-2ea9860d5a46)
![tr3](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/054d68a6-5c5e-4776-8594-daf228caf296)

**Reply by sreedharm07** [reply, marked as answer]
Hi @aambadkar 
spell mistake here, please change it

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/455120ef-3d17-44ad-8e6b-f6f98b120c3c)

**Reply by aambadkar** [reply]
ohh... it was silly mistake. thank you.
